libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyManager.java

```diff
@@ -194,30 +194,45 @@ private String operationDescription(String methodName) {
         return methodName.substring(methodName.indexOf('$'));
     }
 
-    public void checkNetworkAccess(Class<?> callerClass, int actions) {
+    public void checkInboundNetworkAccess(Class<?> callerClass) {
+        checkEntitlementPresent(callerClass, InboundNetworkEntitlement.class);
+    }
+
+    public void checkOutboundNetworkAccess(Class<?> callerClass) {
+        checkEntitlementPresent(callerClass, OutboundNetworkEntitlement.class);
+    }
+
+    public void checkAllNetworkAccess(Class<?> callerClass) {
```

Comment:
I was expecting this to look like the following; is there a reason it needs to be more complex? If so, perhaps it could benefit from a clarifying comment.

```
checkEntitlementPresent(callerClass, InboundNetworkEntitlement.class);
checkEntitlementPresent(callerClass, InboundNetworkEntitlement.class);
```

Reply:
@prdoyle updated your commented to check both entitlements ... but agree, this would be easier to read, despite requiring a few additional calls

Reply:
Skip some duplication (find class, check if trivially allowed); I'm fine either way: I think this is not much more complicated, and checking network access may be on a hot path. But the saving is maybe very little.

Reply:
I don't have a strong opinion, fine either way